CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ For more information about this file see also [Keep a Changelog](http://keepacha
 ## Unreleased
 
 ### Fixes
+- Removed google maps and replaced with leaflet #2105
```

Comment:
would also mention the add site functionality

Reply:
done